docs/framework/windows-workflow-foundation/samples/index.md

```diff
@@ -4,23 +4,25 @@ ms.date: "03/30/2017"
 ms.assetid: e0aef017-4381-4f2f-a904-6e1ed898687e
 ---
 # Windows Workflow (WF) Samples
-These samples provide instruction on various aspects of Windows Workflow Foundation (WF).  
-  
-The samples can be downloaded from the [Microsoft Download Center](https://go.microsoft.com/fwlink/?LinkId=150780).  
-  
+
+These samples provide instruction on various aspects of Windows Workflow Foundation (WF).
+
+The samples can be downloaded from the [Microsoft Download Center](https://go.microsoft.com/fwlink/?LinkId=150780).
+
 > [!NOTE]
->  These samples were created with [!INCLUDE[vs2010](../../../../includes/vs2010-md.md)] and [!INCLUDE[netfx40_short](../../../../includes/netfx40-short-md.md)], but are compatible with [!INCLUDE[vs_current_long](../../../../includes/vs-current-long-md.md)] and [!INCLUDE[netfx_current_short](../../../../includes/netfx-current-short-md.md)]. Additional samples for Windows Workflow Foundation in [!INCLUDE[netfx_current_short](../../../../includes/netfx-current-short-md.md)] can be found on [Code MSDN](https://aka.ms/WF45Samples).
-  
-## In this section  
- [Application](../../../../docs/framework/windows-workflow-foundation/samples/application.md)  
- Provides samples that are related to workflow applications.  
-  
- [Basic](../../../../docs/framework/windows-workflow-foundation/samples/basic.md)  
- Provides samples that demonstrate basic Windows Workflow Foundation (WF) functionality.  
-  
- [Scenario](../../../../docs/framework/windows-workflow-foundation/samples/scenario.md)  
- Provides examples of Windows Workflow Foundation (WF) scenarios.  
-  
- The samples contained in this section make use of new features in C# 3.0. For details on some new features of C# 3.0 that may be unfamiliar, please see the [C# 3.0 Overview](https://go.microsoft.com/fwlink/?LinkId=193972).  
-  
+> These samples were created with Visual Studio 2010 and [!INCLUDE[netfx40_short](../../../../includes/netfx40-short-md.md)], but are compatible with later versions of Visual Studio and the .NET Framework. Additional samples for Windows Workflow Foundation in [!INCLUDE[netfx_current_short](../../../../includes/netfx-current-short-md.md)] can be found on [Code MSDN](https://aka.ms/WF45Samples).
```

Comment:
unrelated to your PR, instead of Code MSDN, it would be good to describe it as MSDN code samples.